app/src/main/java/com/midisheetmusic/SheetMusicActivity.java

```diff
@@ -140,99 +141,89 @@ public void onCreate(Bundle state) {
     /* Create the MidiPlayer and Piano views */
     void createViews() {
         layout = findViewById(R.id.sheet_content);
-
-        SwitchDrawerItem scrollVertically = new SwitchDrawerItem()
-                .withName(R.string.scroll_vertically)
-                .withChecked(options.scrollVert)
-                .withOnCheckedChangeListener((iDrawerItem, compoundButton, isChecked) -> {
-                    options.scrollVert = isChecked;
-                    createSheetMusic(options);
-                });
+        drawer = findViewById(R.id.sheet_content_drawer);
+        drawerLayout = findViewById(R.id.sheet_content_drawer_layout);
+
+        SwitchDrawerItem scrollVertically = new SwitchDrawerItem();
+        scrollVertically.setName(new StringHolder(R.string.scroll_vertically));
+        scrollVertically.setChecked(options.scrollVert);
+        scrollVertically.setOnCheckedChangeListener((iDrawerItem, compoundButton, isChecked) -> {
+            options.scrollVert = isChecked;
+            createSheetMusic(options);
+        });
 
         SwitchDrawerItem colorAccidentals = new SwitchDrawerItem();
 
-        SwitchDrawerItem useColors = new SwitchDrawerItem()
-                .withName(R.string.use_note_colors)
-                .withChecked(options.useColors)
-                .withOnCheckedChangeListener((iDrawerItem, compoundButton, isChecked) -> {
-                    if (isChecked == true)
-                    {
-                        options.colorAccidentals = false;
-                    }
-                    colorAccidentals.setChecked(options.colorAccidentals);
-                    drawer.updateItem(colorAccidentals);
-                    options.useColors = isChecked;
-                    createSheetMusic(options);
-                });
-
+        SwitchDrawerItem useColors = new SwitchDrawerItem();
+        useColors.setName(new StringHolder(R.string.use_note_colors));
+        useColors.setChecked(options.useColors);
+        useColors.setOnCheckedChangeListener((iDrawerItem, compoundButton, isChecked) -> {
+            if (isChecked)
+                options.colorAccidentals = false;
```

Comment:
Please keep the parenthesis even for one-line if statements and loops. I know this might sound redundant if you're a Java/C++ developer, but at my work we have a strict policy to keep them as we have had bugs especially by Python programmers who are used to using indentations for inner code